resources/js/components/fieldtypes/bard/BardFieldtype.vue

```diff
@@ -602,6 +572,46 @@ export default {
             return buttons.filter(button => this.buttonIsVisible(button));
         },
 
+        initEditor() {
+            if (this.editor) this.editor.destroy();
+
+            const content = this.valueToContent(clone(this.value));
+
+            this.editor = new Editor({
+                extensions: this.getExtensions(),
+                content: content,
+                editable: !this.readOnly,
+                enableInputRules: this.config.enable_input_rules,
+                enablePasteRules: this.config.enable_paste_rules,
+                onFocus: () => this.$emit('focus'),
+                onBlur: () => {
+                    // Since clicking into a field inside a set would also trigger a blur, we can't just emit the
+                    // blur event immediately. We need to make sure that the newly focused element is outside
+                    // of Bard. We use a timeout because activeElement only exists after the blur event.
+                    setTimeout(() => {
+                        if (!this.$refs.container.contains(document.activeElement)) {
```

Comment:
This is the only line that was also changed amongst this. `this.$el` was changed to `this.$refs.container`.